package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "@google/maps": "^0.5.5",
+    "@lesswrong/lesswrong-editor": "file:public/ckeditor5-build-balloon-block",
```

Comment:
Does this do anything? It feels like an ontology error to have this in package.json.

Reply:
Nope, that is indeed how we deal with the CKEditor dependency. I tried just importing the file directly, but gave up after many hours of hopeless trying.

Reply:
In general CKEditor recommends and assumes that you will run your editor package as a separate npm package